src/app/common/transactions/requests.spec.ts

```diff
@@ -3,7 +3,7 @@ import { STX_TRANSFER_TX_REQUEST, TEST_WALLET } from '@tests/mocks';
 import { generateContractCallToken } from '@tests/utils/transation-test-utils';
 import { UNAUTHORIZED_TX_REQUEST, verifyTxRequest } from './requests';
 
-describe('verifyTxRequest', () => {
+describe.skip('verifyTxRequest', () => {
```

Comment:
Do you mean to skip this still? Just pointing out in case you intended to remove this.